src/status_im/accounts/create/core.cljs

```diff
@@ -194,9 +195,13 @@
                  :key-code nil
                  :step :confirm-code)}))
 
+;; TODO -- eliminate circular dependency that makes this necessary
+(defn accounts-recover-recover-account [passphrase password]
+  ((resolve 'status-im.accounts.recover.core/recover-account!) passphrase password))
```

Comment:
This looks quite weird, the aforementioned circular dependency should be eliminated.

Reply:
Done!